boards/BMS/Inc/App/App_Accumulator.h

```diff
@@ -15,6 +15,7 @@
 #define MIN_CELL_CHARGE_TEMP_DEGC (0.0f)
 #define MAX_CELL_VOLTAGE (4.2f)
 #define MIN_CELL_VOLTAGE (3.0f)
+#define C_RATE_TO_AMPS (17.7f)
```

Comment:
Nit: It's good to explain where you got this number for constants like these

Reply:
Sounds good, I added the explanation in the ".h" file. I'll also mention it in my write-up.